docs/pages/installation/minikube.md

```diff
@@ -59,16 +59,26 @@ Perform the following steps in order:
 
 ### 2. Install Meshery
 
-Follow the [installation steps](/guides/mesheryctl) to setup the mesheryctl CLI and install Meshery.
+Follow the [installation steps](/installation/quick-start) to setup the mesheryctl CLI and install Meshery.
 
-**Note**: If you are using docker driver, after completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
+**Users using docker driver**: 
+
+After completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
  docker network connect bridge meshery_meshery_1
  </div></div>
  </pre>
 
+To establish connectivity between a particular Meshery Adapter and Kubernetes server, use *"docker ps"* to identify the name of the desired container, and execute the following command:
+
+<pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ docker network connect bridge <container name of the desired adapter>
+ </div></div>
+ </pre>
+
```

Comment:
```cosole
docker network connect bridge <container name of the desired adapter>
```
@leecalcote how about this format? looks more compact to me

Reply:
Oh, the html tags are included so that convenient "copy to clipboard" button is shown to the user and that they can copy and paste with ease. Is that what you're referring to?

Reply:
Yeah, Thanks for explaining :+1: